scripts/exregional_run_anal.sh

```diff
@@ -0,0 +1,968 @@
+The executable specified in exec_fp does not exist:
+  exec_fp = \"${exec_fp}\"
+Build lightning process and rerun."
+fi
+#
+#-----------------------------------------------------------------------
+#
+# Set and export variables.
+#
+#-----------------------------------------------------------------------
+#
+
+#
+#-----------------------------------------------------------------------
+#
+# Run the GSI.  Note that we have to launch the forecast from
+# the current cycle's run directory because the GSI executable will look
+# for input files in the current directory.
+#
+#-----------------------------------------------------------------------
+#
+# comment out for testing
+PREP_STEP
+eval $RUN_CMD_UTILS ${exec_fp} < gsiparm.anl ${REDIRECT_OUT_ERR} || print_err_msg_exit "\
+Call to executable to run GSI returned with nonzero exit code."
+POST_STEP
+
+if [ ${GSI_TYPE} == "ANALYSIS" ]; then
+  if [ ${OB_TYPE} == "radardbz" ]; then
+    cat fort.238 > $COMOUT/${NET}.t${HH}z.fits3.${POST_OUTPUT_DOMAIN_NAME}
+  else
+    mv fort.207 fit_rad1
+    sed -e 's/   asm all/ps asm 900/; s/   rej all/ps rej 900/; s/   mon all/ps mon 900/' fort.201 > fit_p1
+    sed -e 's/   asm all/uv asm 900/; s/   rej all/uv rej 900/; s/   mon all/uv mon 900/' fort.202 > fit_w1
+    sed -e 's/   asm all/ t asm 900/; s/   rej all/ t rej 900/; s/   mon all/ t mon 900/' fort.203 > fit_t1
+    sed -e 's/   asm all/ q asm 900/; s/   rej all/ q rej 900/; s/   mon all/ q mon 900/' fort.204 > fit_q1
+    sed -e 's/   asm all/pw asm 900/; s/   rej all/pw rej 900/; s/   mon all/pw mon 900/' fort.205 > fit_pw1
+    sed -e 's/   asm all/rw asm 900/; s/   rej all/rw rej 900/; s/   mon all/rw mon 900/' fort.209 > fit_rw1
+
+    cat fit_p1 fit_w1 fit_t1 fit_q1 fit_pw1 fit_rad1 fit_rw1 > $COMOUT/${NET}.t${HH}z.fits.${POST_OUTPUT_DOMAIN_NAME}
+    cat fort.208 fort.210 fort.212 fort.213 fort.220 > $COMOUT/${NET}.t${HH}z.fits2.${POST_OUTPUT_DOMAIN_NAME}
+  fi
+fi
+#
+#-----------------------------------------------------------------------
+#
+# touch a file "gsi_complete.txt" after the successful GSI run. This is to inform
+# the successful analysis for the EnKF recentering
+#
+#-----------------------------------------------------------------------
+#
+touch gsi_complete.txt
+#
+#-----------------------------------------------------------------------
+#
+# Copy analysis results to INPUT for model forecast.
+#
+#-----------------------------------------------------------------------
+#
+#
+#if [ ${BKTYPE} -eq 1 ]; then  # cold start, put analysis back to current INPUT 
+#  cp_vrfy ${DATA}/fv3_dynvars                  ${bkpath}/gfs_data.tile7.halo0.nc
+#  cp_vrfy ${DATA}/fv3_sfcdata                  ${bkpath}/sfc_data.tile7.halo0.nc
+#else                          # cycling
+#  cp_vrfy ${DATA}/fv3_dynvars             ${bkpath}/fv_core.res.tile1.nc
+#  cp_vrfy ${DATA}/fv3_tracer              ${bkpath}/fv_tracer.res.tile1.nc
+#  cp_vrfy ${DATA}/fv3_sfcdata             ${bkpath}/sfc_data.nc
+#fi
```

Comment:
Looking at the [exregional_run_anal.sh](https://github.com/NOAA-GSL/regional_workflow/blob/feature/RRFS_dev1/scripts/exregional_run_analysis.sh) script in NOAA-GSL/regional_workflow/RRFS_dev1, I see that this section is also commented out.  Should this section be removed, or perhaps add in a new flag to turn on/off the ability to copy analysis results to INPUT for model forecast?

Reply:
I have removed this for now, but will monitor future updates to the script regarding this section.